apps/client/src/features/cuesheet/Cuesheet.module.scss

```diff
@@ -102,6 +103,9 @@ $table-header-font-size: calc(1rem - 3px);
     position: sticky;
     left: 47.5%; // center of the screen, ish
     padding: 0.5rem 0;
+    &:first-letter {
+      text-transform: uppercase;
+    }
```

Comment:
if this does what I'm thinking it does (capitalizes the first letter) `text-transform: capitalize?`

Reply:
Thank you @arymoraes 
If I am correct, capitalise will make uppercase all the first characters in every word, so like
`My Cuesheet Example`

where we are looking for to capitalising only the very first letter to `My cuesheet example`

Reply:
good point, I thought capitalise would just do the first one